jest.config.js

```diff
@@ -19,16 +19,15 @@ module.exports = {
   moduleDirectories: ['node_modules', 'app/node_modules'],
   setupFiles: ['./internals/scripts/CheckBuildsExist.js'],
   setupFilesAfterEnv: ['<rootDir>/app/tests/_setup-tests.ts'],
-  coverageReporters: ['html'],
+  coverageReporters: ['html', 'json-summary'],
   globals: {
     api: true,
     CONFIG: true,
     'ts-jest': {
       tsconfig: 'tsconfig.tests.json',
       diagnostics: {
         ignoreCodes: [
-          6133,
-          2304,
+          6133, 2304,
```

Comment:
Did you mean to move these on the same line?

Reply:
This is owing to prettier change from the other PR